pages/linux/nova.md

```diff
@@ -0,0 +1,28 @@
+# nova
+
+> The OpenStack project that provides a way to provision compute instances (aka virtual servers).
+> More information: <https://docs.openstack.org/nova/latest/>.
+
+- List VMs on current tenant:
+
+`nova list`
+
+- List VMs of all tenants (admin user only):
+
+`nova list --all-tenants`
+
+- Boot a VM on a specific host:
+
+`nova boot --nic net-id={{net_id}} --image {{image_id}} --flavor {{flavor}} --availability-zone nova:{{host_name}} {{vm_name}}`
+
+- Stop a server:
+
+`nova stop {{server}}`
+
+- Start a server:
```

Comment:
I think the start command should be first then the stop command. Can you rearrange them?